lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -36,6 +36,8 @@ import { ProcessDefinitionCloud } from '../models/process-definition-cloud.model
 })
 export class StartProcessCloudComponent implements OnChanges, OnInit {
 
+    MAX_LENGTH: number = 255;
```

Comment:
use the same convention as start task
    static MAX_NAME_LENGTH = 255;

Reply:
ok